src/app/clusters/fan-control-server/fan-control-server.cpp

```diff
@@ -360,80 +372,70 @@ void MatterFanControlClusterServerAttributeChangedCallback(const app::ConcreteAt
     }
     case PercentSetting::Id: {
         DataModel::Nullable<Percent> percentSetting;
+        uint8_t speedMax;
```

Comment:
Nit pick: should be right above `status = SpeedMax::Get(attributePath.mEndpointId, &speedMax);` since that's where we use this variable.